gsa/src/web/pages/scanners/dialog.js

```diff
@@ -185,7 +185,7 @@ const ScannerDialog = ({
 
   const {gmp} = props;
 
-  if (gmp.settings.enableGreenboneSensor) {
+  if (isDefined(gmp) && gmp.settings.enableGreenboneSensor) {
```

Comment:
Is this necessary? If this is needed for tests only, we should make sure that we provide gmp in the test.